common/web/input-processor/src/text/inputProcessor.ts

```diff
@@ -273,6 +273,7 @@ namespace com.keyman.text {
           let totalMass = 0; // Tracks sum of non-error probabilities.
           for(let pair of keyDistribution) {
             if(pair.p < KEYSTROKE_EPSILON) {
+              totalMass += pair.p;
```

Comment:
I'm not entirely sure which issue this is correcting?

Reply:
This helped with investigating the issue and debugging.  It allows me to infer the original key probability from the final prediction.

Without that, the probability was being changed and didn't match what I expected.